<?xml version="1.0"?>
<launch>
    <node pkg="auto_navigation" type="go_rviz" name="multi_goals" output="screen">
        <!-- Distance to change goal -->
        <!-- <param name="stopDistance" value="0.3"/>    -->
        <param name="map_frame_id" value="map"/> 
        <param name="stopWaitingTime" value="2.0"/>   <!-- Time to wait when arriving goal -->
    </node>
</launch>

